fulp_modules/features/ert/code/modsuits.dm

```diff
@@ -0,0 +1,135 @@
+/obj/item/mod/control/pre_equipped/fulp/ert
+	icon = 'fulp_modules/features/ert/icons/modsuit.dmi'
+	worn_icon = 'fulp_modules/features/ert/icons/modsuit_worn.dmi'
+	// see fulpmodules/Z_edits/modsuits.dm
+	icon_path = 'fulp_modules/features/ert/icons/modsuit.dmi'
+	worn_icon_path = 'fulp_modules/features/ert/icons/modsuit_worn.dmi'
+	cell = /obj/item/stock_parts/cell/hyper
+
+/obj/item/mod/control/pre_equipped/fulp/ert/commander/medical
+	theme = /datum/mod_theme/responsory/fulp/medical
+	initial_modules = list(/obj/item/mod/module/storage/large_capacity, /obj/item/mod/module/flashlight, /obj/item/mod/module/health_analyzer, /obj/item/mod/module/injector)
```

Comment:
```suggestion
	initial_modules = list(
		/obj/item/mod/module/storage/large_capacity,
		/obj/item/mod/module/flashlight,
		/obj/item/mod/module/health_analyzer,
		/obj/item/mod/module/injector,
	)
```

Reply:
this is how the tg ones format it but i agree it looks better